README.md

```diff
@@ -386,6 +385,54 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Method Ordering
+
+Method implementations are grouped and ordered with lifecycle methods first, then by class inheritance. A `pragma mark -` is used to separate each group.
```

Comment:
> Having lifecycle methods be “roughly defined” will I think lead to a lot of confusion down the road for developers who rely on this guide.

As we're getting at in [this discussion](https://github.com/NYTimes/objective-c-style-guide/pull/120#discussion_r28987328), we can nail down the definition.

> This pull request also doesn’t lay out any benefits whatsoever to separating them from their natural place in the inheritance hierarchy.

Updated the PR body: https://github.com/NYTimes/objective-c-style-guide/pull/120#issue-70470804

https://github.com/NYTimes/objective-c-style-guide/commit/5470fe9862d81f2e1f9b41d98de2167c89fb65fd clarifies this rule to reduce complexity.